apps/web/app/(app)/[emailAccountId]/stats/RuleStatsChart.tsx

```diff
@@ -76,7 +80,18 @@ export function RuleStatsChart({ dateRange, title }: RuleStatsChartProps) {
       ),
     };
 
-    return { pieChartData: pieData, chartConfig: config };
+    const barConfig: ChartConfig = {
+      executed: {
+        label: "Executed Rules",
+        color: "#006EFF80",
```

Comment:
we have a lot of magic strings like this in the codebase. also for the landing page stuff. ideally we put it into tailwind config. in the shadcn code examples it seems like you can do this.

but if we do need to use a variable, would still be good to put them into a single file and name the colors